docs/API.md

```diff
@@ -1591,7 +1661,7 @@ whose series is 2. Available filters are:
 * promulgated - the charm has been promulgated.
 * provides - interfaces provided by the charm.
 * requires - interfaces required by the charm.
-* series - the charm's series.
+* series - the set of series associated with the charm.
```

Comment:
I don't follow. The endpoint currently accepts 0 or 1 series as parameter values. We now just allow it to accept > 1.
name=foo&series=precise&series=trusty

Existing use cases will work just fine, as will new callers be able to specify additonal series to filter on. This will be something I anticipate juju may use.